e2e/fixtures/monorepo/packages/waku-project/postcss.config.js

```diff
@@ -1,7 +1,6 @@
 /** @type {import('postcss-load-config').Config} */
 export default {
   plugins: {
-    tailwindcss: {},
-    autoprefixer: {},
+    '@tailwindcss/postcss': {},
```

Comment:
would you want to switch to use tailwind's vite plugin instead of postcss? 

https://tailwindcss.com/docs/upgrade-guide#using-vite

it says:

> If you're using Vite, we recommend migrating from the PostCSS plugin to our new dedicated Vite plugin for improved performance and the best developer experience

Reply:
Sure! Would you like to try?

Reply:
Sure!

Reply:
There's two options: `vite.config.ts` and `waku.config.ts`
I'm not sure if both work or either.